project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.2.3
```

Comment:
Why change in sbt.version. Is that just to align to the latest or need for a specific feature in latest version?

Reply:
This was just to align in latest since the version itself was a bit out of date as well.